src/foundations/shape/Shape.types.ts

```diff
@@ -3,7 +3,7 @@ import { TContainerElement, TInteractionElement } from '../../shared/types';
 // TODO: maybe move the hard-coded ones to a separate union type (`TActionElement` maybe?)
 type TShapeElement = TContainerElement | TInteractionElement;
 
-type TShapeBorderRadius = 0 | 4 | 8 | 12 | 16 | 20 | 24 | 'circle' | 'pill';
+type TShapeBorderRadius = 0 | 2 | 4 | 8 | 12 | 16 | 20 | 24 | 'circle' | 'pill';
```

Comment:
we need to discuss this change with the design team, since the design docs do not have a value of `2` defined for a shape border radius

Reply:
That's what I was thinking too. @matthewbirtch can you advice? In the design files https://www.figma.com/file/GF3yQOh00AHiCHHO6al9ZE/Components---Checkboxes-and-Radios the border radius is set to 2px?

Reply:
Yes, we can add 2px to our acceptable border radii since the smallest checkbox need 2px instead of 4px

Reply:
FYI, I added the 2px radius to the zeroheight docs. Noted that it should only be used for very small elements like the small checkbox